README.md

```diff
@@ -17,103 +24,144 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
+You can create one by following the commands below, or you can just do it yourself:
 
 ```shell
-PHOST=`kubectl get node -o=jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}'`
+kubectl apply -f examples/psql/psql-console.yaml
 ```
 
+And then you will find a Pod named `psql-console` running in the Kubernetes, and you can attach to it to execute commands inside the container with the following command:
+
 ```shell
-PPORT=`kubectl get service -n test test-risingwave-amd64-frontend -o=jsonpath='{.spec.ports[0].nodePort}'`
+kubectl exec -it psql-console bash
 ```
 
-Connect to the frontend by `psql` as follows:
+Finally, we can get access to the RisingWave with the `psql` command inside the Pod:
 
 ```shell
-psql -h $PHOST -p $PPORT -d dev
+psql -h risingwave-in-memory -p 4567 -d dev -U root
 ```
 
-## Storage
+## Storages
 
-### S3
+### Memory
 
-We support using AWS S3 as storage. You can use S3 as follows:
+Currently, memory storage is supported for test usage only. We highly discourage you use the memory storage for other purposes. For now, you can enable the memory metadata and object storage with the following configs:
 
-1. Create a `secret` named `cloud-provider-configure` in your namespace.
-```shell
-kubectl create secret generic -n test cloud-provider-configure --from-literal AccessKeyID=XXXXXXX --from-literal SecretAccessKey=YYYYYYY --from-literal Region=ZZZZZ
+```yamlex
+#...
+spec:
+  storages:
+    meta:
+      memory: true
+    object:
+      memory: true
+#...
 ```
 
-2. Create a `bucket` in the AWS Console.
-3. Use the bucket by setting the following fields: 
+### etcd (meta)
+
+We recommend using the [etcd](https://etcd.io/)](https://etcd.io/) to store the metadata. You can specify the connection information of the `etcd` you'd like to use like the following:
+
 ```yamlex
-objectStorage:
-  s3:
-    provider: aws
-    bucket: xxxx #your-bucket-name
+#...
+spec:
+  storages:
+    meta:
+      etcd: 
+        endpoint: risingwave-etcd:2388
+        secret: etcd-credentials      # optional, empty means no authentication 
+#...
 ```
 
+Check the [examples/risingwave-etcd-minio.yaml](./examples/risingwave-etcd-minio.yaml) for how to provision a simple RisingWave with an `etcd` instance as the metadata storage.
 
-## Configuration
+### MinIO
 
-You can get the `risingwave-operator` configuration as follows:
+We support using MinIO as the object storage. Check the [examples/risingwave-etcd-minio.yaml](./examples/risingwave-etcd-minio.yaml) for details. The YAML structure is like the following:
 
-```shell
-kubectl get cm risingwave-operator-controller-manager-config -n risingwave-operator-system -o yaml
+```yamlex
+#...
+spec:
+  storages:
+    object:
+      minio:
+        secret: minio-credentials
+        endpoint: minio-endpoint:2388
+        bucket: hummock001
+#...
 ```
 
-If you edit the `ConfigMap`, please restart the `risingwave-operator` to reload the configuration.
-
-## Monitoring
```

Comment:
Why remove the `Monitoring` part?

Reply:
It's generally not telling the truth, so I think it should be removed for now, especially the service monitor part. We can add it back to the readme after implementing it.